wordpress-mobile/WordPress-Android
# [Reader] Add tracks for text highlight/copy in Post Details body

Fixes #20888 

Add tracks to text highlight and copy in Post Details body WebView. For that, I used the same javascript code used on iOS (https://github.com/wordpress-mobile/WordPress-iOS/pull/23251/files#diff-0c87a62d20f4b5ede4c7274c4e990d865a633b92dfe70d702672ee27191613d9) to be able to listen to the appropriate events and send a message to the Java/Kotlin code.

-----

## To Test:

1. Open Jetpack
2. Make sure data collection is enabled in `App Settings -> Privacy`
3. Go to Reader
4. Go to any post
5. Select some text
6. **Verify** the `reader_article_text_highlighted` event is tracked (use logcat)
7. Tap on `Copy` in the context menu
8. **Verify** the `reader_article_text_copied` event is tracked

Note: the highlight event has a slight delay (1 second) due to a debounce mechanism. This was done to make sure the event is sent usually after the user selects and adjusts the selection and avoid sending multiple events for each selection change. 

-----

## Regression Notes

1. Potential unintended areas of impact

    - N/A

10. What I did to test those areas of impact (or what existing automated tests I relied on)

    - N/A

11. What automated tests I added (or what prevented me from doing so)

    - Added unit tests for tracking

-----

## PR Submission Checklist:


-----

## Testing Checklist (strike-out the not-applying and unnecessary ones):
